src/discounts/components/SaleDetailsPage/SaleDetailsPage.tsx

```diff
@@ -194,78 +199,40 @@ const SaleDetailsPage: React.FC<SaleDetailsPageProps> = ({
                 <CardSpacer />
                 <TabContainer>
                   <CategoriesTab
+                    testId="categories-tab"
                     isActive={activeTab === SaleDetailsPageTab.categories}
                     changeTab={onTabClick}
                   >
-                    {intl.formatMessage(
-                      {
-                        id: "ppLwx3",
-                        defaultMessage: "Categories ({quantity})",
-                        description: "number of categories",
-                      },
-                      {
-                        quantity: maybe(
-                          () => sale.categories.totalCount.toString(),
-                          "…",
-                        ),
-                      },
-                    )}
+                    {intl.formatMessage(itemsQuantityMessages.categories, {
+                      quantity: tabItemsCount.categories?.toString() || "…",
```

Comment:
`0` would fall back to `...`
```suggestion
                      quantity: tabItemsCount.categories?.toString() ?? "…",
```
Same thing happens multiple times below

Reply:
Not really. Indeed, `0` would fall back to `"..."`. But here, it is `0.toString()` which means `"0"` which fall back to `"0"`.